documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+|    var2 = init2                  |         var2 = init2 then step2,      |
+|    while not test:               |         until: test)                  |
+|      body                        |      body                             |
+|      var1 = step1                |    finally result                     |
+|      var2 = step2                |    end for                            |
+|    return result                 |                                       |
++----------------------------------+---------------------------------------+
+
+Predefined functions
+--------------------
+
+Boolean functions
+^^^^^^^^^^^^^^^^^
++--------------------------------+-----------------------------------------------+
+| Python                         | Dylan                                         |
++================================+===============================================+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    not obj                     |    ~ obj                                      |
+|                                |    ~obj                                       |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    isinstance(obj, bool)       |    instance?(obj, <boolean>)                  |
++--------------------------------+-----------------------------------------------+
+
+Equivalence predicates
+^^^^^^^^^^^^^^^^^^^^^^
++--------------------------------+-----------------------------------------------+
+| Python                         | Dylan                                         |
++================================+===============================================+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    x == y                      |    x == y                                     |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | N/A                                           |
+|                                |                                               |
+|    x is y                      |                                               |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    x == y                      |    x = y                                      |
++--------------------------------+-----------------------------------------------+
+
+Pairs and Lists
+^^^^^^^^^^^^^^^
```

Comment:
TBH a lot of this section doesn't feel very helpful for Pythonistas. They won't really be looking for a linked list implementation and some of the equivalences feel a bit contrived.  It would work better in a "Python for Dylaners" document.